assets/third_party/tiny-swords/Terrain/Water/Rocks/Rocks_01.png.import

```diff
@@ -1,34 +0,0 @@
-[remap]
```

Comment:
Did you mean to rename one of the Tiny Swords rocks? I don't think so – I'll move it back.